cli/transform.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Turn an array of strings of form `key=value` into an object of form `{ key: value }`
+ */
+export function parseFilters(filter: string[]): Record<string, unknown> {
+  return filter.reduce((acc, curr) => {
+    const [key, value] = curr.split("=");
+    return { ...acc, [key]: value };
+  }, {});
+}
+
+/**
+ * Remove keys from a `Record` whose value is falsy
+ */
+export function removeUndefined(
+  obj: Record<string, string | undefined>,
+): Record<string, string> {
+  return Object.entries(obj).filter(([, value]) => !!value).reduce(
```

Comment:
I think this would stop you for filtering for any values that are 0?

```
!!0 == false
!!1 == true
```

Reply:
You're correct, however `removeUndefined` takes a string, and `!!"0"` is `true`... because JavaScript..

<img width="282" alt="image" src="https://github.com/guardian/devx-logs/assets/836140/910ce3d4-cdb3-41ec-854d-74094cffcea0">

I've added a unit test for this now too - 824038db8ba5d877ebe1b0c09ae6c2a994978ba7.